styles/dataDashboard.css

```diff
@@ -236,28 +258,24 @@ textarea {
         flex-direction: column;
     }
 
-
     .title{
         display: flex;
         flex-direction: column;
     }
-
-    .row {
-        display: flex;
-        flex-direction: column;
-        flex-wrap: wrap;
-    }
 }
 
-@media only screen and (min-width: 768px),
-       only screen and (min-width: 700px) and (orientation: landscape) {
+@media only screen and (min-width: 865px) {
         
     .main-header {
             display: flex;
             justify-content: flex-end;
             align-items: center;
             padding: 30px 10%;
     }
+  
+    .main-content {
+        padding-left: 3rem;
+    }
```

Comment:
Larger viewport. Add back the padding that was removed above for small viewports.